tests/DependencyInjection/SonataAdminExtensionTest.php

```diff
@@ -42,6 +42,27 @@ protected function setUp(): void
         ;
     }
 
+    public function testHasCoreServicesAlias(): void
+    {
+        $this->assertContainerBuilderHasService('Sonata\AdminBundle\Admin\Pool');
+        $this->assertContainerBuilderHasService('Sonata\AdminBundle\Route\AdminPoolLoader');
+        $this->assertContainerBuilderHasService('Sonata\AdminBundle\Admin\AdminHelper');
+        $this->assertContainerBuilderHasService('Sonata\AdminBundle\Filter\FilterFactory');
+        $this->assertContainerBuilderHasService('Sonata\AdminBundle\Admin\BreadcrumbsBuilder');
+        $this->assertContainerBuilderHasService('Sonata\AdminBundle\Translator\BCLabelTranslatorStrategy');
+        $this->assertContainerBuilderHasService('Sonata\AdminBundle\Translator\NativeLabelTranslatorStrategy');
+        $this->assertContainerBuilderHasService('Sonata\AdminBundle\Translator\LabelTranslatorStrategyInterface');
```

Comment:
Please, add a check against the type returned by this service.

Reply:
Done